x/validator/handler.go

```diff
@@ -106,12 +106,15 @@ func handleMsgWithdrawReward(ctx sdk.Context, keeper Keeper, msg MsgWithdrawRewa
 	if !found {
 		return nil, fmt.Errorf("Reward does not exist")
 	}
-	if !reward.HasNewReward() {
-		logEntry.Warn = append(logEntry.Warn, "no new reward")
-		return &sdk.Result{}, nil
+
+	if reward.LastWithdrawTime.Add(keeper.WithdrawWindow(ctx)).After(ctx.BlockTime()) {
+		if !reward.HasNewReward() {
+			logEntry.Warn = append(logEntry.Warn, "no new reward")
+			return &sdk.Result{}, nil
+		}
```

Comment:
InitateWithdraw should be triggered even there is no new reward, in case the signer voting power changed

Reply:
never mind, I was confused